test/lib/import/brca/providers/st_george_old/st_george_handler_old_test.rb

```diff
@@ -10,6 +10,18 @@ def setup
     @logger = Import::Log.get_logger
   end
 
+  test 'process_fields' do
+    e_batch = EBatch.create(original_filename: 'test_file',
+                            e_type:            'PSMOLE',
+                            provider:          'RJ7_2',
+                            registryid:        'RJ7_2')
+    handler = Import::Brca::Providers::StGeorgeOld::StGeorgeHandlerOld.new(e_batch)
+    handler.process_fields(@record)
+    assert_difference('EBatch.count', 1) do
+      handler.finalize
+    end
```

Comment:
worth checking the e_batch has been updated to `RJ7` here?

Reply:
Added in [latest commit](https://github.com/NHSDigital/data_management_system/pull/86/commits/639f01c26c1b91d6aa3987f8e0bb97f94003e3cb)